3rd_party/CMakeLists.txt

```diff
@@ -32,12 +32,13 @@ set(ENTITYX_DT_TYPE double CACHE INTERNAL "")
 set(ENTITYX_RUN_BENCHMARKS false CACHE INTERNAL "")
 set(ENTITYX_MAX_COMPONENTS 64 CACHE INTERNAL "")
 
-add_subdirectory(dbopl)
-add_subdirectory(entityx)
-add_subdirectory(glad)
-add_subdirectory(speex_resampler)
-add_subdirectory(stb)
-add_subdirectory(imgui)
-add_subdirectory(imgui-filebrowser)
+# EXCLUDE_FROM_ALL to disable installing these HO or static library deps
+add_subdirectory(dbopl EXCLUDE_FROM_ALL)
```

Comment:
since I added installation steps to the project, I wanted to make sure we don't install all the unnecessary development files (headers, static libs, etc...)

Reply:
Makes sense - I remember running into this the last time I experimented with CPack, because the `entityx` library also has its own `install` steps that ended up adding some unneeded header files to the package.

I wasn't aware of this `EXCLUDE_FROM_ALL` flag, that's a very nice solution!